preflight/pkg/controller/cache.go

```
@@ -0,0 +1,235 @@
+
+		return []corev1.Volume{cached}
+	}
+
+	return nil
+}
+
+// stubTypedPod reduces a pod outside any preflight-enabled namespace to the
+// minimum identity fields needed by the informer to track the object.
+func stubTypedPod(pod *corev1.Pod) *corev1.Pod {
+	pod.ObjectMeta = metav1.ObjectMeta{
+		Name:            pod.ObjectMeta.Name,
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "ObjectMeta" from selector

(staticcheck)